crypto/x509/x_crl.c

```diff
@@ -103,13 +103,17 @@ static int crl_set_issuers(X509_CRL *crl)
 
         if (gtmp) {
```

Comment:
`gtmp != NULL` ?

Reply:
This is a bug-fix patch.  Therefore I'd like to keep it as small as possible so it will still cherry-pick to 1.1.1